### ISSUE TITLE
LibreSSL 3.5.x make check fails

### ISSUE THREAD

**Issue by mledford** [libressl-portable/portable#744]
I am trying to compile 3.5.1 on macOS and make check fails. I have tried on macOS Big Sur on an M1 machine as well as Catalina on Intel. The failure is `asn1object` for `i2t_ASN1_OBJECT()`. In hopes of quickly figuring it out I added a few outputs in the test portion that was failing but it looks like it is a bit deeper than that. I have checked 3.5.0 and see the same thing however 3.4.3 passes. Below is the output from the test that I modified.

```
==========================================
   libressl 3.5.1: tests/test-suite.log
==========================================

# TOTAL: 99
# PASS:  98
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: asn1object
================

FAIL: i2t_ASN1_OBJECT() failed
aobj = 0x7f9874c05bd0
aobj->data = 0x7f9874c05c00
ret = 0
sizeof(buf) = 1024
FAIL: i2t_ASN1_OBJECT() failed
aobj = 0x7f9874c05bd0
aobj->data = 0x7f9874c05c00
ret = 0
sizeof(buf) = 1024
FAIL asn1object (exit status: 1)
```

**Comment by botovq** [comment]
There is an unfortunate use of uninitialized that didn't make it into 3.5.1.
Pretty sure you're running into that one:

https://github.com/libressl-portable/openbsd/commit/f0812c1af482df6ee636f92fb5fcfb4734b04a85

It will be in 3.5.2 to be released shortly.

**Comment by mledford** [comment]
Thank you for the information! I will keep a lookout for 3.5.2 and let you know if it is resolved.

**Comment by busterb** [comment]
3.5.2 release is coming soon, I'm going to close this issue for now.

**Comment by mledford** [comment]
The failure listed above has been resolved in 3.5.2. However, a new failure has appeared in 3.5.2. I will create a new ticket for that.
